parser-typechecker/src/Unison/Util/Pretty.hs

```diff
@@ -341,6 +354,13 @@ numbered
   -> Pretty s
 numbered num ps = column2 (fmap num [1 ..] `zip` toList ps)
 
+numberedColumn2
+  :: (Foldable f, LL.ListLike s Char, IsString s)
+  => (Int -> Pretty s)
+  -> f (Pretty s, Pretty s)
+  -> Pretty s
+numberedColumn2 num ps = numbered num (align $ toList ps)
```

Comment:
How is this different from `numbered`?  Maybe an example in comment?

Reply:
It's like `column2`, but with numbers next to each.

Reply:
added an example in the docs and explanation